dynamicPermissions.groups.ps1

```diff
@@ -0,0 +1,456 @@
+######################################################
+# HelloID-Conn-Prov-Target-HelloID-Dynamic-Permissions-Groups
```

Comment:
Recommend changing dynamicPermissions to subPermissions

Suggestion also put resource/permission files in folders

Reply:
Changed.